examples/discordjs/src/index.js

```diff
@@ -1,21 +1,58 @@
-const { Client, Intents } = require("discord.js");
+const { REST } = require('@discordjs/rest'); // Define REST.
+const { Routes } = require('discord-api-types/v9'); // Define Routes.
+const fs = require('fs'); // Define fs (file system).
+const { Client, Intents, Collection } = require("discord.js"); // Define Client, Intents, and Collection.
 const client = new Client({
   intents: [Intents.FLAGS.GUILDS, Intents.FLAGS.GUILD_MESSAGES]
-});
+}); // Make || Connect to a new discord bot client.
```

Comment:
What does || in this context mean?

Reply:
"or"

Reply:
So that means.. Make or connect to a new discord bot client..?

Reply:
Fwiw, constructing that Client class doesn't make one.
```suggestion
}); // Connect to our discord bot.
```